### ISSUE TITLE
SDK Exemplar class missing Data suffix

### ISSUE THREAD

**Issue by anuraaga** [open-telemetry/opentelemetry-java#3535]
We add Data suffix to all the SDK data classes, so I guess Exemplar should have it too. Noticed when writing a marshaler for it, it's the only class that requires an explicit import of the proto since the proto name and SDK name are the same.

**Comment by jkwatson** [comment]
@jsuereth can I assign to you?

**Comment by jsuereth** [comment]
@jkwatson yes, sorry somehow I lost bug notifications for a few weeks their, updated my GitHub settings
